### ISSUE TITLE
Use tools.build

### ISSUE THREAD

**Issue by olttwa** [jlesquembre/clojars-publish-action#3]
### Problem
`clojars-publish-action` uses [depstar](https://github.com/seancorfield/depstar) to build a JAR. However, depstar has been archived by @seancorfield

### Solution
We can use `tools.build` as that's the standard going ahead.

**Comment by jlesquembre** [comment]
Agree, I didn't updated the project for a long time, I'll try to find time to do that
